bchrpc/server.go

```diff
@@ -527,7 +541,7 @@ func (s *GrpcServer) GetRawBlock(ctx context.Context, req *pb.GetRawBlockRequest
 	} else {
 		h, err := chainhash.NewHash(req.GetHash())
 		if err != nil {
-			return nil, status.Error(codes.InvalidArgument, "invalid hash")
+			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("invalid hash %s", err.Error()))
```

Comment:
A few more down here. Just is traditionally done as `error message: <val>` in error strings.

Reply:
done, I think :)